recipes/nnetsauce/meta.yaml

```diff
@@ -0,0 +1,50 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "nnetsauce" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/b2/89/8fa407c95cf2f89c7b79b3ad43c5517382b1628861d6e640376f3d9c53b0/{{ name|lower }}-{{ version }}-cp38-cp38-macosx_10_15_x86_64.whl
+  sha256: 0f82123ffe1b4678f21f2efa79023b254355cd8cb8eb13cef75d7badca54f0eb
+  #url: https://github.com/Techtonique/{{ name|lower }}/tree/master/dist/{{ name|lower }}-{{ version }}.tar.gz
```

Comment:
When I calculated the sha256 of the release download I get the following:

```
$ openssl sha256 ~/Downloads/nnetsauce-0.8.2.tar.gz
SHA256(/Users/xyz/Downloads/nnetsauce-0.8.2.tar.gz)= cfce4e3b0e6b4d76ab2ac37ba7727aa4fdbb747164a3022afef15b216d755b66
```

There is no release for 0.8.3 that I see (https://github.com/Techtonique/nnetsauce/releases)

Reply:
Been struggling with this, when I'm 
testing  locally. Is it just:
`openssl sha256 ~/Downloads/nnetsauce-0.8.2.tar.gz`?
A new sha256 seems to be pop up each 
and everytime I try.